core/domain/topic_domain.py

```diff
@@ -1132,7 +1134,8 @@ def validate(self, strict=False):
 
     @classmethod
     def create_default_topic(
-            cls, topic_id, name, url_fragment, description):
+            cls, topic_id: str, name: str, url_fragment: str, description: str
+            ) -> Topic:
```

Comment:
Remove the typing and add ignore to the callsite

Reply:
done!